client/client_test.go

```diff
@@ -614,51 +612,6 @@ func TestConcurrentIncrements(t *testing.T) {
 	}
 }
 
-// TestClientPermissions verifies permission enforcement.
```

Comment:
it doesn't check anything per method anymore, it's purely a `node` user vs others. If you switch the `TestHTTPAuthentication` test in `kv/db_test.go` to use rpc, then we'll be covered.